Microsoft.Toolkit.Services/Services/LinkedIn/LinkedInDataProvider.cs

```diff
@@ -86,18 +90,35 @@ public LinkedInDataProvider(LinkedInOAuthTokens tokens, LinkedInPermissions requ
         }
 #endif
 
+#if WINFORMS
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LinkedInDataProvider"/> class.
+        /// Constructor.
+        /// </summary>
+        /// <param name="tokens">OAuth tokens for request.</param>
+        /// <param name="requiredPermissions">Required permissions for the session.</param>
+        public LinkedInDataProvider(LinkedInOAuthTokens tokens, LinkedInPermissions requiredPermissions)
```

Comment:
No validation required on params?

Reply:
Right, tokens and requiredPermissions should be validated here and in the other constructors